bitcoingold/0.15.2/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" == "bgold-cli" || "$1" == "bitcoin-tx" || "$1" == "bgoldd" || "$1" == "test_bitcoin" ]]; then
+	mkdir -p "$BITCOIN_DATA"
+
+	cat <<-EOF > "$BITCOIN_DATA/bitcoin.conf"
```

Comment:
`bitcoingold.conf`?

Though Bitcoin Gold itself accepts `bitocoin.conf`, it's mainly for backward compatibility. It will always try to rename `bitcoin.conf` to `bitcoingold.conf`.

Reply:
PR updated